pkg/controllers/namespace/namespace_controller.go

```diff
@@ -211,30 +208,19 @@ func (r *NamespaceReconciler) Reconcile(ctx context.Context, req ctrl.Request) (
 			log.Error(err, "failed to build namespace and network config bindings", "Namepspace", ns)
 			return common.ResultRequeueAfter10sec, nil
 		}
-		// read anno "nsx.vmware.com/vpc_name", if ns contains this annotation, it means it will share
-		// infra VPC, if the ns in the annotation is the same as ns event, create infra VPC, if not,
-		// skip the event.
+		// read anno "nsx.vmware.com/vpc_name", if ns contains this annotation, it means it will share infra VPC
 		ncName, ncExist := annotations[types.AnnotationVPCNetworkConfig]
 		vpcName, nameExist := annotations[types.AnnotationVPCName]
-		var createVpcName *string
 		if nameExist {
-			log.Info("read ns annotation vpcName", "VPCNAME", vpcName)
+			log.Info("read ns annotation networkInfoName", "VPCNAME", vpcName)
 			res := strings.Split(vpcName, "/")
 			// The format should be namespace/vpc_name
 			if len(res) != 2 {
-				message := fmt.Sprintf("incorrect vpcName annotation %s for namespace %s", vpcName, ns)
+				message := fmt.Sprintf("incorrect networkInfoName annotation %s for namespace %s", vpcName, ns)
```

Comment:
VPC name, the tag is now used for nsx vpc, it is not related to network info.

Reply:
fixed